www/src/components/repository/RepositoryEdit.tsx

```diff
@@ -331,7 +381,28 @@ function RepositoryEdit() {
         heading="Edit"
         paddingTop="medium"
       >
-        <Flex display-desktop-up="none"><RepositoryActions /></Flex>
+        <Flex
+          align="center"
+          gap="medium"
+        >
+          {formHasUpdates && (
+            <P
+              body2
+              color="text-xlight"
+            >
+              Unsaved changes
+            </P>
+          )}
+          <Button
+            type="submit"
+            enabled={submitEnabled}
+            disabled={!submitEnabled}
+            loading={loading}
+          >
+            Update
+          </Button>
+          <Flex display-desktop-up="none"><RepositoryActions /></Flex>
+        </Flex>
```

Comment:
There's a `components/utils/SaveButton` component you could use here. I should probably move it to design system. I think it's also supposed to say 'Save' now instead of 'Update.'